<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistertionForm</class>
 <widget class="QWidget" name="RegistertionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-image: url(:/ing/LoginPageImg.png);
background-repeat :no-repeat;
background-position: center;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="UsernameEnter">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>110</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>160</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>220</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Re-Enter Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="PasswordEnter">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>170</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="PasswordReEnter">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>230</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Sigup">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background:none;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
        <widget class="QPushButton" name="GoLogin">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>360</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background:none;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
        <widget class="QLabel" name="WrongPass">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>260</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>305</x>
           <y>230</y>
           <width>24</width>
           <height>24</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
qproperty-icon:url(:/ing/pngwing.com.png);
qproperty-iconSize: 35px 35px;
border: 0px solid</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
